clubs/README.md

```diff
@@ -174,35 +190,194 @@ keep building their website to make it unique. At the second meeting, you'll
+importantly_, get them excited about the club.
+
+The beginning of the previously mentioned slideshow presentation contains slides
+to run with the introduction presentation.
+
+#### Name Game
+
+The name game is a simple icebreaker activity that will warm the atmosphere and
+make club members more comfortable around each other. This will be especially
+helpful to freshman, who probably don't know people from the higher grades.
+
+Have the club members gather in large circle. Start by saying your name and a
+mnemonic for it (ex. "Hey everyone, I'm Zach. Zany Zach."). Then have the person
+to your left do the same and then repeat what you said (ex. "I'm Jolly Jane. And
+he is Zany Zach"). Then have the person to their left do the same and then say
+the names and mnemonics for everyone that has gone so far, in order. Continue
+doing this through the whole circle until everyone has gone. This will get
+significantly more difficult (and fun!) as more people go -- if someone gets
+completely stuck, encourage others to help them out.
+
+#### First Website Workshop
+
+To get members started, you'll want to run the
+[Portfolio](../workshops/portfolio/README.md) workshop. It's going to walk club
+members through the process of building their first website. We recommended
+doing the following to run the Portfolio workshop (and we've prepared slides, in
+the previously linked slideshow presentation to help run it).
+
+1. Tell everyone that to start out, we're going to run the Portfolio workshop,
+   which will walk them through the process of building their first website
+2. Show the slide with the output of the Portfolio workshop and tell everyone
+   that this is what they'll build by the end of the workshop, which will take
+   around 45 minutes
+3. Say that after the Portfolio workshop, the rest of the meeting will be
+   completely free-form for them to continue building their websites. Show the
+   slides with examples of websites other people in Hack Clubs have made to give
+   your members ideas for directions they can take their sites. Say that at the
+   end of the next meeting, everyone is going to come to the front and demo what
+   they built.
+
+#### Free-Form Hacking
+
+After hackers finish the Portfolio workshop, they're going to transition into
+free-form hacking to continue building their websites. During this time, the
+most important thing for you to do is create an environment where people are
+comfortable asking for help.
+
+When people do have questions, be eager to help them out. It's also good idea to
+turn the music back on (not too loud, people should feel comfortable talking).
+
+#### Second Meeting Reminder
+
+Five minutes before the meeting is over, get everyone's attention to remind them
+about the next meeting. Say when and where it'll be and that everyone will be
+finishing the websites they started on and presenting them at the end in it. And
+thank them for coming to this meeting!
```

Comment:
It exists in the sections for the second meeting.

On Mon, Jan 18, 2016 at 12:11 PM, Jonathan Leung <notifications@github.com>
wrote:

> In clubs/README.md
> <https://github.com/hackclub/hackclub/pull/657#discussion_r50039508>:
>
> > +#### Free-Form Hacking
> > +
> > +After hackers finish the Portfolio workshop, they're going to transition into
> > +free-form hacking to continue building their websites. During this time, the
> > +most important thing for you to do is create an environment where people are
> > +comfortable asking for help.
> > +
> > +When people do have questions, be eager to help them out. It's also good idea to
> > +turn the music back on (not too loud, people should feel comfortable talking).
> > +
> > +#### Second Meeting Reminder
> > +
> > +Five minutes before the meeting is over, get everyone's attention to remind them
> > +about the next meeting. Say when and where it'll be and that everyone will be
> > +finishing the websites they started on and presenting them at the end in it. And
> > +thank them for coming to this meeting!
>
> You need to create a #### Demos section and talk about how to do demos
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/657/files#r50039508>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>